features/steps/room_steps.js

```diff
@@ -8,37 +8,42 @@ const assert = require('assert').strict;
 const { By, until } = require('selenium-webdriver');
 const RoomSettingPage = require("../page-objects/RoomSettingPage");
 
+/**
+ * Merges extracted parameter types together for convenience within step definitions
+ */
```

Comment:
This could use more comments explaining why this function makes things convenient and an example of the ideal way to invoke it.

Reply:
Yea, I think In need to up my jsdoc fluency, because I was like "ehhhh, I don't want to figure out how to annotate this function definition!" even though I _knew_ I would want an example and information about what parameters it was expecting and why.

Reply:
https://jsdoc.app/tags-example.html seems helpful. Tho I personally prefer adding an empty new line in between examples and long sentences for easy scannability and readability.